plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -325,6 +325,34 @@ public void testCaseInsensitive()
         assertUpdate("DROP TABLE testcase.testinsensitive");
     }
 
+    @Test
+    public void testNonLowercaseViewName()
+    {
+        // Case insensitive schema name
+        MongoCollection<Document> collection = client.getDatabase("NonLowercaseSchema").getCollection("test_collection");
+        collection.insertOne(new Document(ImmutableMap.of("Name", "abc", "Value", 1)));
+
+        client.getDatabase("NonLowercaseSchema").createView("lowercase_view", "test_collection", ImmutableList.of());
+        assertQuery("SELECT value FROM nonlowercaseschema.lowercase_view WHERE name = 'abc'", "SELECT 1");
+
+        // Case insensitive view name
```

Comment:
nit: This comment is misleading. The view name is already lowercase.

Reply:
I think it's not lowercase or am I missing something? 

L342 `... createView("NonLowercaseView" ...`

Reply:
Sorry, I was looking at the collection name. Thanks. 😄